lic12.cpp does not compile

lic12.cpp uses the assert function without including `cassert`

This leads to compilation error.

Full error message:
```
% make
g++ -I. -std=c++2a   -c -o decide/lic12.o decide/lic12.cpp
decide/lic12.cpp: In function ‘bool lic12(Points, Parameters)’:
decide/lic12.cpp:52:3: error: ‘assert’ was not declared in this scope
   52 |   assert(0 <= length2);
      |   ^~~~~~
decide/lic12.cpp:2:1: note: ‘assert’ is defined in header ‘<cassert>’; did you forget to ‘#include <cassert>’?
    1 | #include <decide/decide.hpp>
  +++ |+#include <cassert>
    2 |
make: *** [<builtin>: decide/lic12.o] Error 1
```

--- a/decide/lic12.cpp
+++ b/decide/lic12.cpp
@@ -1,5 +1,5 @@
-#include <decide/decide.hpp>
 #include <cassert>
+#include <decide/decide.hpp>
 
 // There exists at least one set of two data points, separated by exactly K_PTS
 // consecutive intervening points, which are a distance greater than the length,

--- a/decide/lic12.cpp
+++ b/decide/lic12.cpp
@@ -1,4 +1,5 @@
 #include <decide/decide.hpp>
+#include <cassert>
 
 // There exists at least one set of two data points, separated by exactly K_PTS
 // consecutive intervening points, which are a distance greater than the length,

--- a/decide/lic12.test.cpp
+++ b/decide/lic12.test.cpp
@@ -9,7 +9,7 @@
 void test_true_instance() {
   Points points{Coordinate(1, 1), Coordinate(2, 2), Coordinate(3, 3),
                 Coordinate(4, 4), Coordinate(5, 5), Coordinate(100, 100)};
-  Parameters parameters{.LENGTH1 = 50, .LENGTH2 = 6, .KPTS = 2};
+  Parameters parameters{.LENGTH1 = 50, .KPTS = 2, .LENGTH2 = 6};
   assert(lic12(points, parameters));
 }
 
@@ -19,7 +19,7 @@ void test_true_instance() {
 void test_false_instance1() {
   Points points{Coordinate(1, 1), Coordinate(2, 2), Coordinate(3, 3),
                 Coordinate(4, 4), Coordinate(5, 5), Coordinate(100, 100)};
-  Parameters parameters{.LENGTH1 = 500, .LENGTH2 = 6, .KPTS = 2};
+  Parameters parameters{.LENGTH1 = 500, .KPTS = 2, .LENGTH2 = 6};
   assert(!lic12(points, parameters));
 }
 
@@ -29,7 +29,7 @@ void test_false_instance1() {
 void test_false_instance2() {
   Points points{Coordinate(1, 1), Coordinate(2, 2), Coordinate(3, 3),
                 Coordinate(4, 4), Coordinate(5, 5), Coordinate(100, 100)};
-  Parameters parameters{.LENGTH1 = 50, .LENGTH2 = 1, .KPTS = 2};
+  Parameters parameters{.LENGTH1 = 50, .KPTS = 2, .LENGTH2 = 1};
   assert(!lic12(points, parameters));
 }
 

